_includes/code/graphql.additional.py

```diff
@@ -1,37 +1,41 @@
-import weaviate
-import json
+# TEST DISABLED - sandbox needed
 
-client = weaviate.Client(
-    'https://edu-demo.weaviate.network',
-    auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
-)
+# import weaviate
+# import json
+
+# client = weaviate.Client(
+#     'https://edu-demo.weaviate.network',
+#     auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
+# )
 
 
-def test_gqlresponse(response_in, gqlresponse_in):
-    for i, result in enumerate(response_in['data']['Get']['JeopardyQuestion']):
-        assert result['question'] == gqlresponse_in['data']['Get']['JeopardyQuestion'][i]['question']
+# def test_gqlresponse(response_in, gqlresponse_in):
+#     for i, result in enumerate(response_in['data']['Get']['JeopardyQuestion']):
+#         assert result['question'] == gqlresponse_in['data']['Get']['JeopardyQuestion'][i]['question']
 
 
 # ===================
 # ===== Sorting =====
 # ===================
 # START Sorting Python
-response = (
-    client.query
-    .get('JeopardyQuestion', ['question', 'answer'])
+from weaviate.collections.classes.grpc import Sort
+  
+article=client.collections.get("JeopardyQuestion")
+response = article.query.fetch_objects(
     # highlight-start
-    .with_sort({
-        'path': ['answer'],
-        'order': 'asc',
-    })
+    sort=Sort(prop="answer", ascending=True),
     # highlight-end
-    .with_limit(3)
-    .do()
-)
+    limit=3
+)  
 
-print(json.dumps(response, indent=2))
+for o in response.objects:
```

Comment:
I think it's a bit busy myself, but nbd. 

The idiomatic expressions are different because with the new client you have to actively choose to print certain properties, whereas that would be more work with the v3 client.